CHANGELOG.md

```diff
@@ -44,6 +44,7 @@ deprecated and all components removed except the `legacy/` package. This require
 genesis state. Namely, `accounts` now exist under `app_state.auth.accounts`. The corresponding migration
 logic has been implemented for v0.38 target version. Applications can migrate via:
 `$ {appd} migrate v0.38 genesis.json`.
+* (modules) [\#5249](https://github.com/cosmos/cosmos-sdk/pull/5249) Funds are now allowed to be directly sent to the community pool (via the distribution module account).
```

Comment:
@RiccardoM apologies, but I realized these changes only affect the `SimApp` -- the state machine does not change. i.e. this PR doesn't need any changelog entry. Can you undo this change please?